FrontEnd/src/components/basics/Fallback/Fallback.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from 'react';
+import { amenityToIcon, Amenity } from '../Amenities/Amenities';
+import { Icon as IconType, miscIcons } from '@icons';
+
+// Amenity needs to be any so it can be type guarded
+interface FallbackProps {
+  amenity: any;
+}
+
+// Type guard to check if key is of Amenity type
+const isAmenity = (key: any): key is Amenity => {
+  return key in amenityToIcon;
+};
+
+// Fallback Icon Component (could be a normal amenity svg too)
+const FallbackIcon: FunctionComponent<FallbackProps> = ({ amenity }) => {
+  const Icon: IconType = isAmenity(amenity)
```

Comment:
Can shorten the function to the following:
```
const FallbackIcon: FunctionComponent<FallbackProps> = ({ amenity }): IconType => (
  isAmenity(amenity)
    ? amenityToIcon[amenity]
    : miscIcons.Fallback
);
```

Reply:
Storybook wasn't liking this change for some reason, so I kept it how it was originally